test/linter/test-notes.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+const chalk = require('chalk');
+const parser = require('node-html-parser');
+const { HtmlValidate } = require('html-validate');
+const { VALID_ELEMENTS } = require('../utils.js');
```

Comment:
`VALID_ELEMENTS` is only used in this lint rule, can we just put it here and not in utils? It seems unlikely we'll use it anywhere else.